src/main/resources/ManifestFile.txt

```diff
@@ -1,4 +1 @@
-AuthorPageConfig
-AuthorPageConfigTest
-SampleClass2
```

Comment:
Its good to have Non existent class(es) in the manifest file. Tests a functionality(Skipping non-existent test classes and informing users)